Parser/pgen/pgen.py

```diff
@@ -27,8 +27,8 @@
   rule_name: rule_description;
 
   meaning the rule 'a: b' specifies that a can be replaced by b. A Context-free
```

Comment:
Same as https://github.com/python/cpython/pull/15414.
```suggestion
  meaning the rule 'a: b' specifies that a can be replaced by b. A context-free
```

Reply:
I don't know how to update a PR. That's why I closed #15414 and created a new updated one.

Reply:
You need to push more commits to your branch. If you are not familiar with git I can do it for you

Reply:
Thanks. I figured out that pushing more commits to my branch and creating a pull request afterward would've done the trick. I'm still learning how to work with GitHub. :)

Reply:
Notice that this PR is created from your master branch, so you need to put more commits there and push to your remote branch for them to appear here.

Check out https://realpython.com/python-git-github-intro/ for how to do that and what these terms mean

Reply:
I'm still using it through the website rather than the terminal. So this `remote branch` was indeed confusing to me. I checked that out. Seems like I've got a lot more to explore and learn. Thanks for the direction.

Reply:
You can also hit the "apply code suggestion" from this web page and later pull the changes on your local branch to have this changes reflected.

Reply:
@aw1621107 Thanks! Committed that.